packages/block-library/src/gallery/edit.js

```diff
@@ -220,7 +222,7 @@ class GalleryEdit extends Component {
 				<Fragment>
 					{ controls }
 					<MediaPlaceholder
-						icon="format-gallery"
+						icon=<BlockIcon icon={ icon } />
```

Comment:
Can it follow other definitions of props?:
```suggestion
						icon={ <BlockIcon icon={ icon } /> }
```